include/wx/settings.h

```diff
@@ -140,7 +140,11 @@ enum wxSystemMetric
     wxSYS_DCLICK_MSEC,
     wxSYS_CARET_ON_MSEC,
     wxSYS_CARET_OFF_MSEC,
-    wxSYS_CARET_TIMEOUT_MSEC
+    wxSYS_CARET_TIMEOUT_MSEC,
+    wxSYS_DEVICE_AREA_TOP,
+    wxSYS_DEVICE_AREA_BOTTOM,
+    wxSYS_DEVICE_AREA_LEFT,
+    wxSYS_DEVICE_AREA_RIGHT
```

Comment:
We really should add this to `wxDisplay` if it's not already covered by its `GetClientArea()` and not here, this is basically a legacy interface.